src/Controls/tests/SourceGen.UnitTests/InitializeComponent/CompiledBindings.cs

```diff
@@ -299,4 +299,70 @@ static IEnumerable<string> SplitCode(string code)
 				.Select(static line => line.Trim())
 				.Where(static line => !string.IsNullOrWhiteSpace(line));
 	}
+
+	[Theory]
+	[InlineData(".")]
+	[InlineData("")]
```

Comment:
Is there a valid way in XAML to express the self-referencing binding with explicilty stating Path? Is `"{Binding Path=.}"` valid? It seems it should be. Make sure that input is also one if the input data (please pass the full `"{Binding ...}"` in the InlineData value instead of deriving it later in code)